msteams-platform/toolkit/AAD-manifest-customization.md

```diff
@@ -0,0 +1,312 @@
+---
```

Comment:
Seems like this doc didn't get added into the ToC yet?

Reply:
Created a placeholder for this doc, it is visible in the review site now, once it gets merged then it will be visible in the doc site as well.